tests/ShopBundle/Functional/Model/Pricing/InputPriceRecalculationSchedulerTest.php

```diff
@@ -14,20 +13,17 @@
 use Shopsys\FrameworkBundle\Model\Pricing\Vat\VatData;
 use Shopsys\FrameworkBundle\Model\Product\Availability\Availability;
 use Shopsys\FrameworkBundle\Model\Product\Availability\AvailabilityData;
-use Shopsys\FrameworkBundle\Model\Product\ProductFacade;
 use Shopsys\FrameworkBundle\Model\Transport\TransportDataFactoryInterface;
 use Shopsys\FrameworkBundle\Model\Transport\TransportFacade;
-use Shopsys\ShopBundle\Model\Product\Product;
-use Shopsys\ShopBundle\Model\Product\ProductDataFactory;
 use Symfony\Component\HttpKernel\Event\FilterResponseEvent;
 use Tests\ShopBundle\Test\TransactionFunctionalTestCase;
 
 class InputPriceRecalculationSchedulerTest extends TransactionFunctionalTestCase
 {
     public function testOnKernelResponseNoAction()
     {
+        /** @var \Shopsys\FrameworkBundle\Component\Setting\Setting $setting */
```

Comment:
I wonder, how come that in some cases, you fixed these annotations? :laughing:

Reply:
Yes, when I rewrote Tests, I tried to unify annotations, so it's easier to read a code. 
This should be unified (#29)